indico/modules/events/controllers/display.py

```diff
@@ -84,8 +84,11 @@ class RHDisplayPrivacyPolicy(RHEventBase):
     """Display event privacy policy."""
 
     def _process(self):
-        if urls := privacy_settings.get(self.event, 'privacy_policy_urls'):
-            # There could be multiple urls, redirect to the first one.
-            return redirect(urls[0]['url'])
+        if self.event.type_ == EventType.conference:
+            return WPConferencePrivacyDisplay.render_template(
+                'display/privacy.html',
+                self.event,
+                privacy_info=privacy_settings.get_all(self.event)
+            )
         return WPDisplayPrivacyPolicy.render_template('privacy.html',
```

Comment:
I only noticed now, but this WP and template are coming from the `legal` module which is meant for the global privacy notice... so it's rendered completely unrelated to the event. Since we no longer have a footer link pointing to this page in any case, I think we can always render the conference version - so if someone appends `/privacy` to the URL in a meeting they just get it in the conference layout like it happens for most other conference-only pages as well...